### PR TITLE
create PR template

### PR DESCRIPTION
**Pull request by bradfordcondon**
PR template as discussed in #390

### CODE REVIEW

**Comment by bradfordcondon** [comment]
we might want to make an analogous PR to 7.x-2.x?

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] API-specific change (fix or addition to an API function)
+- [ ] Updates documentation (inline or markdown files)
+
+## Description
+<!--- Describe your changes in detail -->
+<!--- Why is this change required? What problem does it solve? -->
+<!--- If it fixes an open issue, please link to the issue here. -->
```
Comment (laceysanderson) [inline PULL_REQUEST_TEMPLATE.md]: > `<!--- If it fixes an open issue, please link to the issue here. -->`

This is duplicated (we moved it to the top).

File: PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+<!--- Provide a general summary of your changes in the Title above -->
+<!--- See our Contribution Guidelines here:
+          https://github.com/tripal/tripal/blob/7.x-3.x/CONTRIBUTING.md -->
```
Comment (laceysanderson) [inline PULL_REQUEST_TEMPLATE.md:3]: Maybe add an empty line here so that `Issue #` doesn't get missed?

Comment (bradfordcondon) [reply PULL_REQUEST_TEMPLATE.md:3]: agreed, added

File: PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (bradfordcondon) [reply PULL_REQUEST_TEMPLATE.md]: thanks, removed

**Comment by laceysanderson** [comment]
Awesome :-) Approved! In the issue consensus was this only needs one review (if any) so I say we merge it 🎉

**Comment by bradfordcondon** [comment]
great!  Something like this isn't going to break the code and can always be re-discussed, so let's do it.

Thanks!

**Comment by laceysanderson** [comment]
Done! 🎉

**Pull request merged**